### PR TITLE
Rewrite driver launch

### PR DESCRIPTION
**Pull request by KmakD**

### CODE REVIEW

File: panther_driver/launch/driver.launch [hunk 1]
```diff
@@ -1,37 +1,15 @@
...
-        <include file="$(find panther_driver)/launch/panther_driver.launch">
-            <arg name="robot_width" value="0.697"/>
-            <arg name="wheel_radius" value="0.1825"/>
-            <arg name="kinematics_type" value="differential"/>
-            <arg name="kalman_params" value="$(arg kalman_params)"/>
-        </include>
-    </group>
+  <!-- launch driver node -->
+  <node pkg="panther_driver" name="panther_driver" type="driver_node.py" output="screen" required="true">
+    <rosparam command="load" file="$(find panther_bringup)/config/panther_common.yaml" />
```
Comment (Kotochleb) [inline panther_driver/launch/driver.launch]: Why there is assumption panther driver is cloned alongside panther_ros?

File: panther_driver/launch/driver.launch [hunk 2]
```diff
@@ -1,37 +1,15 @@
...
+    <rosparam command="load" file="$(arg wheel_config_file)" />
```
Comment (Kotochleb) [inline panther_driver/launch/driver.launch]:
```suggestion
```

File: panther_driver/launch/imu.launch [hunk 3]
```diff
@@ -4,21 +4,15 @@
...
+    args="$(optenv PANTHER_IMU_LOCALIZATION_X 0.168) 
+    $(optenv PANTHER_IMU_LOCALIZATION_Y 0.028) 
+    $(optenv PANTHER_IMU_LOCALIZATION_Z 0.083) 
+    $(optenv PANTHER_IMU_ORIENTATION_R 3.14) 
+    $(optenv PANTHER_IMU_ORIENTATION_P -1.57) 
+    $(optenv PANTHER_IMU_ORIENTATION_Y 0.0) 
+    base_link imu_link 100"
+  />
 
   #### Nodelet manager ######################################################
```
Comment (Kotochleb) [inline panther_driver/launch/imu.launch]: We could move IMU parameters to YAML as well I guess

**Review by Kotochleb — CHANGES_REQUESTED** [review]

File: panther_driver/launch/driver.launch (hunk 1 above)
Comment (KmakD) [reply panther_driver/launch/driver.launch]: In my opinion `panther_common.yaml` should be accessible from `panther_bringup` package, so we need another agr I guess. Or arg with default value from panther_driver that can be overriden from panther_bringup?

File: panther_driver/launch/imu.launch (hunk 3 above)
Comment (KmakD) [reply panther_driver/launch/imu.launch]: 2 separate files for ImuFilterNodelet and PhidgetsSpatialNodelet?

File: panther_driver/launch/driver.launch (hunk 1 above)
Comment (Kotochleb) [reply panther_driver/launch/driver.launch]: The only problem is that in this case we assume user only has panther driver downloaded. That is the main problem. In panther_ros we can move it somewhere else

**Review by Kotochleb — DISMISSED** [review]

**Review by pkowalsk1 — APPROVED** [review]

**Pull request merged**